src/pages/TeacherDashboard/TeacherDashboard.jsx

```diff
@@ -1,63 +1,92 @@
+                setCourses(data);
+            }).catch(error => {
+                console.log(error);
+                navigate('/');
+            });
     }, []);
 
+    const totalPoints = (lessons) => {
+        return lessons.reduce((pointSum, {points}) => pointSum + points, 0);
+    };
+
+    if (!token) return null;
+
     return (
-        <>
-            <SideBar/>
-            <div>
-                <h1>Welcome to TeacherDashboard</h1>
-            </div>
+        <div className={styles.teacherDashboardContainer}>
+            <SideBar />
             <div className={styles.courseContainer}>
+                <div className={styles.headerDashboard}>
+                    <h1>Welcome to TeacherDashboard</h1>
+                </div>
                 <table className={styles.courseTable}>
-                    <thead>
-                        <tr>
-                            <th>Course</th>
-                            <th>Total lessons</th>
-                            <th>Points</th>
-                            <th>Enrolled students</th>
-                        </tr>
+                    <thead className={styles.courseTableHeader}>
+                    <tr>
+                        <th></th>
+                        <th>Course</th>
+                        <th>Total lessons</th>
+                        <th>Points</th>
+                        <th>Enrolled students</th>
+                    </tr>
                     </thead>
-                    <tbody>
-                        {
-                            courses.map((course, index) => {
+                    <tbody className={styles.courseTableBody}>
+                    {
+                        courses.length === 0 ?
+                            <tr>
+                                <td colSpan="5">You don’t have any courses. To start your way as a teacher add a new
+                                    course today
+                                </td>
+                            </tr> :
+                            courses.map((course) => {
                                 return (
-                                    <tr key={index}>
+                                    <tr key={course.id}>
+                                        <td><img className={styles.courseLogo} src={course.imageUrl}/></td>
                                         <td>{course.courseName}</td>
                                         <td>{course.lessons.length}</td>
-                                        <td>{course.lessons.reduce((pointSum, {points}) => pointSum + points, 0)}</td>
+                                        <td>{totalPoints(course.lessons)}</td>
                                         <td>{course.user_courses.length}</td>
                                     </tr>
                                 )
                             })
-                        }
+                    }
                     </tbody>
                 </table>
+                <div>
```

Comment:
This doesn't need to be in a div if it's only one item.

Reply:
I prefer to leave it for easy styling

Reply:
I see that you are using this to center the button in the flex container. You could also give `alignItems: center` to the container to accomplish the same thing.

Reply:
Didn't work for the table view